src/components/Menu/index.jsx

```diff
@@ -1,20 +1,95 @@
-import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+
 import styles from './styles.module.css';
 
-const Menu = () => (
-  <nav className={styles.nav}>
-    <ul className={styles.ul}>
-      <li>
-        <Link to="/">Home</Link>
-      </li>
-      <li>
-        <Link to="/exhibitions/">Exhibitions</Link>
-      </li>
-      <li>
-        <Link to="/about/">About</Link>
-      </li>
-    </ul>
-  </nav>
-);
+const pages = [
+  <NavLink to="/" className={({ isActive }) => classNames(styles.link, { [styles.activeLink]: isActive })}>
+    Home
+  </NavLink>,
+  <NavLink to="/exhibitions/" className={({ isActive }) => classNames(styles.link, { [styles.activeLink]: isActive })}>
+    Exhibitions
+  </NavLink>,
+  <NavLink to="/about/" className={({ isActive }) => classNames(styles.link, { [styles.activeLink]: isActive })}>
+    About
+  </NavLink>,
+];
+
+const MenuComponent = () => {
+  const [anchorElNav, setAnchorElNav] = useState(null);
+
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  return (
+    <AppBar position="static">
+      <Container maxWidth="x1">
+        <Toolbar disableGutters>
+          <Box className={styles.buttonBar}>
+            {pages.map((page) => (
+              <Button key={page} onClick={handleCloseNavMenu}>
```

Comment:
You cannot pass objects like `page` as a value to the `key` prop. Please use scalar values instead. As we discussed the other day, if you do it like it's right now, then you'll get this React warning in the console:

```
Encountered two children with the same key, `[object Object]`
```